[2023-07-19T20:41:15.446+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.transform_load_data manual__2023-07-19T20:41:10.078951+00:00 [queued]>
[2023-07-19T20:41:15.681+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_stock_data_google_DAG.transform_load_data manual__2023-07-19T20:41:10.078951+00:00 [queued]>
[2023-07-19T20:41:15.684+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-07-19T20:41:15.707+0000] {taskinstance.py:1327} INFO - Executing <Task(BashOperator): transform_load_data> on 2023-07-19 20:41:10.078951+00:00
[2023-07-19T20:41:15.712+0000] {standard_task_runner.py:57} INFO - Started process 267 to run task
[2023-07-19T20:41:15.722+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'ETL_stock_data_google_DAG', 'transform_load_data', 'manual__2023-07-19T20:41:10.078951+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/ETL_stock_data_google_DAG.py', '--cfg-path', '/tmp/tmpnlirq4o4']
[2023-07-19T20:41:15.726+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask transform_load_data
[2023-07-19T20:41:15.817+0000] {task_command.py:410} INFO - Running <TaskInstance: ETL_stock_data_google_DAG.transform_load_data manual__2023-07-19T20:41:10.078951+00:00 [running]> on host 90317fb99301
[2023-07-19T20:41:15.931+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='fvinciguerra' AIRFLOW_CTX_DAG_ID='ETL_stock_data_google_DAG' AIRFLOW_CTX_TASK_ID='transform_load_data' AIRFLOW_CTX_EXECUTION_DATE='2023-07-19T20:41:10.078951+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-19T20:41:10.078951+00:00'
[2023-07-19T20:41:15.932+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-07-19T20:41:15.935+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /opt/***/dags/pyscripts/TransformationLoad_stock_data_google.py']
[2023-07-19T20:41:15.948+0000] {subprocess.py:86} INFO - Output:
[2023-07-19T20:41:19.459+0000] {subprocess.py:93} INFO - 2023-07-19 20:41:19,459 - ERROR - Error: [Errno 2] No such file or directory: '/opt/***/dags/pyscripts/data/extracted_data.csv'
[2023-07-19T20:41:19.642+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-07-19T20:41:19.699+0000] {taskinstance.py:1345} INFO - Marking task as SUCCESS. dag_id=ETL_stock_data_google_DAG, task_id=transform_load_data, execution_date=20230719T204110, start_date=20230719T204115, end_date=20230719T204119
[2023-07-19T20:41:19.796+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-07-19T20:41:19.824+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
